rotkehlchen/balances/manual.py

```diff
@@ -32,24 +37,37 @@ class ManuallyTrackedBalanceWithValue(NamedTuple):
     location: Location
     tags: list[str] | None
     balance_type: BalanceType
+    asset_is_missing: bool = False
```

Comment:
What are you doing here? Why add it here and add it also in `ManuallyTrackedBalanceWithFlags`. That should be an attribute only of `ManuallyTrackedBalanceWithFlags` not of this. And you should just extend the serialize over there, not touch it here. 

Basic inheritance concepts.


Otherwise if you want to add the `asset_is_missing` to the basic class then just do that. You then don't need the other class.